main/src/main/java/SimpleNotePad.java

```diff
@@ -0,0 +1,344 @@
+import sun.nio.cs.StandardCharsets;
+
+import java.awt.Dimension;
+import java.awt.Graphics;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Properties;
+import javax.swing.JFileChooser;
+import javax.swing.JFrame;
+import javax.swing.JMenu;
+import javax.swing.JMenuBar;
+import javax.swing.JMenuItem;
+import javax.swing.JOptionPane;
+import javax.swing.JScrollPane;
+import javax.swing.JTextPane;
+import javax.swing.text.Position;
+import javax.swing.text.StyledDocument;
+
+public class SimpleNotePad extends JFrame{
```

Comment:
Hey Michael, this is still a really long method. I would consider pulling parts of it out into either different methods or different classes.